documentation/webpack.config.js

```diff
@@ -147,5 +150,5 @@ module.exports = merge(openWcConfig, {
             },
         ],
     },
-    plugins: [new WebpackBar()],
+    plugins: [new WebpackBar(), new BundleAnalyzerPlugin()],
```

Comment:
Added this, but need to confirm the way to conditionally include so it doesn't block on the CI...